app/src/main/java/org/oppia/app/home/topiclist/PromotedStoryListAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.app.home.topiclist
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.oppia.app.databinding.PromotedStoryCardBinding
+
+/** Adapter to inflate different items/views inside [RecyclerView]. The itemList consists of various ViewModels. */
+class PromotedStoryListAdapter(
```

Comment:
Is it necessary to create a separate adapter? I feel like using BindableAdapter would suffice.

Reply:
Sorry, I didn't read the explanation correctly.

Reply:
No issues, I will keep this comment for now, and will solve this.